tasks/main.yml

```diff
@@ -27,7 +27,7 @@
 - name: main | install mongos package
   apt:
     name="mongodb-org-mongos"
-    state="present"
+    state="latest"
```

Comment:
Shouldn't we use a specific number to match what we are using currently, which is `3.4.6`?

Reply:
No apt does not work like that

Reply:
You can specify the version with `sudo apt-get install package=version`. But maybe that's not a good idea?

Reply:
Not many options are usually there, however I've just checked and mongo repository indeed has lots of different versions available.